File: 백준/Bronze/6825. Body Mass Index/Body Mass Index.c
#include <stdio.h>

int main(void) { 
   float a, b;
   scanf("%f", &a);
   scanf("%f", &b);

   float BMI = a / (b * b);
   if(BMI > 25){
      printf("Overweight\n");
   }
   
   else if(BMI > 18.5 && BMI <= 25.0){
      printf("Normal weight\n");
   }

   else{
      printf("Underweight\n");
   }
   
   return 0;
}

File: 백준/Bronze/6825. Body Mass Index/Body Mass Index.py
a = float(input())
b = float(input())

BMI = a / (b * b)

if(BMI > 25):
    print("Overweight")

elif(BMI > 18.5 and BMI <= 25.0):
    print("Normal weight")
    
else:
    print("Underweight")